extensions/git/src/repository.ts

```diff
@@ -785,6 +794,16 @@ export class Repository implements Disposable {
 		this._onDidChangeStatus.fire();
 	}
 
+	private detectActiveFile(fullFilePath: string): boolean {
+		if (window.activeTextEditor !== undefined) {
```

Comment:
Why not just:

return window.activeTextEditor && window.activeTextEditor.document.fileName === fullFilePath;

Reply:
Thank you for the suggestion, that does look nicer. 
I updated my pull request here: #36183